src/SingleCellAnnotator.imjoy.html

```diff
@@ -1457,8 +1495,12 @@
               </div>  
               <div v-for="(v, task_name) in s" :key="task_name">
                 <div class="card-subtitle text-gray">{{task_name}}</div>
+                <div :id="k + task_name">
```

Comment:
Note you have to click to show the charts. I cannot find a way to automatically execute the fun and show the chart directly when loading